quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -304,3 +304,25 @@ impl Limiter for Limits {
         *self
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    // Local max data limits should be <= u32::MAX
```

Comment:
This code comes from #1693 and I'm just moving it, since it doesn't really belong in this module.

To answer your question: we may increase the limit in the future. We also can't break the API so we can't change the size.